src/pybind/mgr/mgr_module.py

```diff
@@ -1386,6 +1386,14 @@ def get_perf_schema(self,
         """
         return self._ceph_get_perf_schema(svc_type, svc_name)
 
+    def get_rocksdb_version(self) -> Dict[str, int]:
+        """
+        Called by the plugin to fetch the latest RocksDB version number.
+
+        :return: dict of the major, minor, and patch RocksDB version numbers
```

Comment:
Refresh comments?

Reply:
Ah, good catch.

Reply:
3rd force-push reflects change to the mgr module description.
Before: `:return: dict of the major, minor, and patch RocksDB version numbers`
After: `:return: dict of the major, minor, and patch RocksDB version numbers`